Unsorted "TODO" tasks:

* lock.py: _ContextManager is used differently in Tulip, is it correct?
* Future.set_exception(): FIXME if debug
* Windows: use _overlapped in py33_winapi?
* Write unit test when yield is used without From()
* Future: only format the traceback in debug mode?
* test_internal_fds() of test_events.py emits two ResourceWarning on Windows
* tests/test_events.py -v KqueueEventLoopTests.test_read_pty_output failed on Mac OS X:
  OSError: [Errno 22] Invalid argument
* Fix tests failing with PyPy:

  - sys.getrefcount()
  - test_queues.test_repr
  - test_futures.test_tb_logger_exception_unretrieved

* python3.3 runtests.sh:
  sys:1: ResourceWarning: gc: 3 uncollectable objects at shutdown; use gc.set_debug(gc.DEBUG_UNCOLLECTABLE) to list them
* write unit test for create_connection(ssl=True)
* Fix examples:

  - stacks.py: 'exceptions.ZeroDivisionError' object has no attribute '__traceback__'

* as_completed() is a coroutine or not?
* Fix all FIXME in the code
* See:
  http://code.google.com/p/appengine-ndb-experiment/source/browse/ndb/tasklets.py
